howsthewaterfront/src/components/SearchResult.js

```diff
@@ -1,7 +1,194 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
+import Search from "./Search";
+import CopyrightFooter from "./Footer";
+import toiletIcon from "../assets/icons8-toilet-50.png";
+import parkingIcon from "../assets/icons8-parking-60.png";
+import wheelchairIcon from "../assets/icons8-wheelchair-48.png";
+import strollerIcon from "../assets/icons8-stroller-50.png";
+import dogIcon from "../assets/icons8-dog-paw-64.png";
+import volleyIcon from "../assets/icons8-volleyball-player-50.png";
+import picnicIcon from "../assets/icons8-picnic-table-50.png";
+import surf_header from "../assets/beach_surf_header.jpg";
+import placeholder01 from "../assets/beach-placeholder01.png";
+import placeholder02 from "../assets/beach-placeholder02.png";
+import placeholder03 from "../assets/beach-placeholder03.png";
+import "../styles/search-result.css";
+
+const beachDummyData = {
+  beachName: "Marina del Rey Harbor",
```

Comment:
I am going to paste the db name for each variable so you have it for when you pull the real data:
beachName = NameMobileWeb